extensions/2.0/Khronos/KHR_animation_pointer/README.md

```diff
@@ -56,7 +56,7 @@ It is not allowed to animate a glTFid property, as it does change the structure
 
 ### Valid target templates
 
-|`path`                                                |Accessor Type|Component Type(s)  |Description                                                   |
+|`pointer`                                             |Accessor Type|Component Type(s)  |Description                                                   |
```

Comment:
Just wanted to note that this (together with a couple more places) was also part of my PR here in case you missed it: 
https://github.com/ux3d/glTF/pull/20

Reply:
Oh, yes, I missed it.